lib/pages/home.dart

```diff
@@ -1,34 +1,132 @@
 import 'package:flutter/material.dart';
-import 'package:flutter_map/flutter_map.dart';
-import 'package:latlong2/latlong.dart';
+import 'package:quickalert/quickalert.dart';
```

Comment:
Have you tries the getWidget library you mentioned? I think we could use it. Then we wouldn't have to find package with every widget we need

Reply:
Yes, I tried it and I'm not actually a big fan of that lib 😅 You were right, it would bring that much value.